src/components/molecules/selectableCardList/SelectableCardList.tsx

```diff
@@ -0,0 +1,32 @@
+import { CheckIcon } from '@/assets/icons';
+import { Typography } from '@/components';
+
+interface SelectableCardListProps {
+  items: {
+    title: string;
+    content: string | React.ReactNode;
+  }[];
```

Comment:
```suggestion
  guideTexts: {
    originalText: string;
    highlightedText: string | React.ReactNode;
  }[];
```
뭔가 엄청 좋은 예시는 아닌 거 같지만 하나는 그냥 원본 텍스트이고 하나는 하이라이트가 포함된 텍스트임을 명시적으로 적으면 좋을 것 같아요!

처음 봤을 때 이 타입을 가지는 객체까지가서 무슨 형태인지 파악해야 했어요 ㅠㅠ

Reply:
해당 component는 재사용하려고 만든 molecules 입니다. 카드에는 text만 들어가는 게 아니라 어떤 컴포넌트가 들어와도 선택 시 값을 처리할 수 있게 만들었어요.
예: content에는 image을 포함한 content가 전달 될 수 있음.

해당 molecule의 사용 방식은 아래와 같습니다.
- title에는 카드 선택 시 상위 컴포넌트에 callback으로 전달할 값을 넣어주세요.
- content에는 카드에 출력할 component를 넣어주세요.

위 설명을 코드에 커맨트로 남겨두었습니다~!